.github/workflows/tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
might have missed the thread, but any particular reason we're dropping 3.8 support? Might as well wait until its EOL in August?

Reply:
> remove python 3.8 from supported versions due to use of asyncio.TaskGroup in server